plugins/discovery-file/src/test/resources/rest-api-spec/test/discovery_file/10_basic.yaml

```diff
@@ -1,13 +1,13 @@
 # Integration tests for file-based discovery
 #
 "Discovery File loaded":
```

Comment:
done in https://github.com/elastic/elasticsearch/pull/20492/commits/c41361d00bb6f4ddeccaa731aea636458be44bf2